### ISSUE TITLE
consider how much time is spent on starting issue, forking and submitting PR

### ISSUE THREAD

**Issue by wikfeldt** [coderefinery/documentation#50]
it's very nice to repeat the forking-PR workflow from git-collab, but we don't want a majority of the time to be spent on this rather than on learning the RTD/Sphinx/RST parts

**Comment by bast** [comment]
The creating an issue can be left out but is the forking and PR really a hurdle? In other words: what do you suggest we do instead?

**Comment by bast** [comment]
It is important to me that participants see where we are heading to before we do all the tweaking. The tweaking can IMO be done "at home". In several workshops I have observed among participants confusion about what they need to do and why and it was not clear why we do it. I have also observed that some participants were not able to do a fork/PR without browsing other lessons but does that not mean that we perhaps failed somewhere else? The previous versions of this lesson were IMO way too much focusing on RST syntax and sometimes missed the big picture. I am totally fine to revert what I did. What I did should have been introduced via a PR and a proper discussion and it wasn't because I did not have time simply. Which is not an excuse. Old approach is in the Git history. Totally fine for me to branch this into two alternative lessons or to go back to previous approach.
